src/styles/blocks/calendar.scss

```diff
@@ -35,8 +36,8 @@
     }
 
     &:hover {
-      cursor: pointer;
       background-color: pink;
+      transition: transform 0.1s;
```

Comment:
**Wrong code.** You need to add a transform property with the correct function. Read the task carefully.